js/ui/scheduler/ui.scheduler.js

```diff
@@ -661,6 +661,22 @@ var Scheduler = Widget.inherit({
                 */
             onAppointmentDblClick: null,
 
+            /**
+                * @name dxSchedulerOptions_onAppointmentContextMenu
+                * @publicName onAppointmentContextMenu
+                * @type function(e)|string
+                * @extends Action
+                * @type_function_param1 e:object
+                * @type_function_param1_field4 appointmentData:object
+                * @type_function_param1_field5 targetedAppointmentData:object
+                * @type_function_param1_field6 appointmentElement:dxElement
+                * @type_function_param1_field7 jQueryEvent:jQuery.Event:deprecated(event)
+                * @type_function_param1_field8 event:event
+                * @type_function_param1_field9 cancel:Boolean
```

Comment:
the "cancel" field is not required as we do not have the built-in context menu, whose showing we may cancel